scanamo/src/main/scala/com/gu/scanamo/query/ConditionExpression.scala

```diff
@@ -58,6 +58,28 @@ object ConditionExpression {
     }
   }
 
+  implicit def symbolValueInCondition[V: DynamoFormat] = new ConditionExpression[(Symbol, Set[V])] {
+    override def apply(pair: (Symbol, Set[V]))(condition: Option[RequestCondition]): RequestCondition = {
+      attributeValueInCondition.apply((AttributeName.of(pair._1), pair._2))(condition)
+    }
+  }
+
+  implicit def attributeValueInCondition[V: DynamoFormat] = new ConditionExpression[(AttributeName, Set[V])] {
+    val prefix = "inCondition"
+    override def apply(pair: (AttributeName, Set[V]))(condition: Option[RequestCondition]): RequestCondition = {
+      val format = DynamoFormat[V]
+      val attributeName = pair._1
+      val attributeValues = pair._2.zipWithIndex.map { case (v, i) => 
+        s":conditionAttributeValue$i" -> format.write(v)
+      }.toMap
+      RequestCondition(
+        s"""#${attributeName.placeholder(prefix)} IN ${attributeValues.keys.mkString("(", ",", ")")}""",
```

Comment:
If `attributeValues.isEmpty` then we'll get a query that checks for an empty string (`attr IN ("")`). Is this the desired behaviour?

Reply:
Yes, that is entirely possible. It is a consequence of using `Set` instead of `NonEmptySet`. I have tried to address this [previously](https://github.com/scanamo/scanamo/pull/2130) but decided to delay this until v1 is out.